app/views/course/levels/index.html.slim

```diff
@@ -1,8 +1,14 @@
 h1 = t('.header')
+
+- if can?(:create, Course::Level)
+  = new_button([current_course, :level]) { t('.create') }
```

Comment:
The idea behind the helper was for it to be consistent. Why do you want to have your own block?